packages/mattermost-redux/src/actions/threads.ts

```diff
@@ -282,6 +274,33 @@ export function markThreadAsUnread(userId: string, teamId: string, threadId: str
     };
 }
 
+export function markLastPostInThreadAsUnread(userId: string, teamId: string, threadId: string) {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const getPostsForThread = makeGetPostsForThread();
+        let posts = getPostsForThread(getState(), threadId);
+
+        const getThreadInfo = getThread(userId, teamId, threadId);
+        const {data: thread} = await getThreadInfo(dispatch, getState);
```

Comment:
I don't think we need to fetch the thread again, since we already have that info.
Is there a case that we need to re-fetch?

Reply:
Probably I am missing something, but I think we dont have thread info (we only have `threadId`). In order to check `thread.reply_count` we need to fetch the thread.

Reply:
Since we are at the threads list, the thread is loaded in redux (we are seeing the reply count on the thread item footer.
You can retrieve it by using the selector `getThread(state, threadId)` (not the action)

Reply:
🤝 now I see your point. You are right. The code is updated as you suggested. 

> The action getThread (which could be better renamed, IMO, fetchThread) is calling the server to fetch the thread again.

In all places, `getThread` is imported as `fetchThread`, so it makes sense to rename it. Please let me know if you want me to rename it as part of this PR. 
One example is below

https://github.com/mattermost/mattermost-webapp/blob/e949d96a76995b7149d7b467074f99bd40968db7/actions/websocket_actions.jsx#L37-L38